src/db/mysql/MySqlAPI.cpp

```diff
@@ -566,6 +566,69 @@ static int getActiveCount(soci::session& sql, const std::string &source, const s
     return activeCount;
 }
 
+/// Determines if the link has a defined upper bound in the 
+// database and returns the number of files allowed for transfer on the specified link
+/// @param source Source storage
+/// @param dest Destination storage
+/// @return bool indicating if there is an upper bound capacity on link
+/// @return int representing maxActive on link
+/// @return int representing currentActive on link
```

Comment:
nit: may need to update comments: three return values listed in comments, two return values actually returned